style.css

```diff
@@ -0,0 +1,408 @@
+  display: inline;
+}
+.bullets-ul{
+  width: 850px;
+  list-style-type: none;
+}
+.bullets-li {
+  display: inline-block;
+  font-family: 'Oswald', sans-serif;
+  font-size: 30px;
+  color: #e0e0e0;
+  border-right: 1px solid #e0e0e0;
+  width: 160px;
+  text-align: justify;
+  padding-left: 20px;
+  padding-right: 20px;
+  box-sizing: border-box;
+}
+.bullets-li:last-child {
+  border-right: none;
+}
+.bullets figure {
+  margin-left: 0;
+}
+
+/*beatles*/
+.beatles {
+  clear: both;
+  padding-top: 120px;
+  margin-left: 100px;
+}
+.beatles-display {
+  display: block;
+  width: 960px;
+}
+.beatles-h1 {
+  font-family: 'Oswald', sans-serif;
+  color: white;
+  font-size: 50px;
+  font-weight: 500;
+  background-color: #524741;
+  text-align: center;
+  margin-bottom: 0;
+  padding: 20px 0;
+  text-transform: uppercase;
+}
+.beatles-figure {
+  margin: 0;
+}
+.beatles-p {
+  font-family: 'Merriweather', serif;
+  font-size: 24px;
+  color: #ffffff;
+  background-color: #524741;
+  text-align: justify;
+  padding: 15px;
+  width: 960px;
+  margin-top: 0;
+  box-sizing: border-box;
+  z-index: 800;
+}
+.photocred {
+  font-size: 16px;
+  opacity: 0.5;
+  padding-bottom: 10px;
+}
+/*immigration*/
+.immigration {
+  width: 680px;
+  margin-left: 100px;
+  float: left;
+}
+.immigration figure {
+  margin: 0;
+  z-index: 700;
+}
+.immigration ul {
+  list-style-type: none;
+  margin: 0;
+  padding-left: 0;
+}
+.immigration li {
+  display: inline-block;
+  width: 160px;
+  padding-right: 5px;
+}
+.immigration h1 {
+  font-family: 'Oswald', sans-serif;
+  font-size: 70px;
+  border-bottom: 10px solid black;
+  text-align: justify;
+  margin: 0;
+  text-transform: uppercase;
+}
+.immigration h3 {
+  font-family: 'Oswald', sans-serif;
+  font-size: 26px;
+  border-bottom: 3px solid black;
+  margin: 0;
+  padding: 5px 0px;
+}
+.immigration p {
+  font-family: 'Lora', serif;
+  text-align: left;
+  font-size: 24px;
+}
+
+/*transformers*/
+.transformers {
+  max-width: 220px;
+  height: 700px;
+  margin: 20px;
+  float: left;
+  color: white;
+}
+.transformers figure {
+  margin-left: 20px;
+  position: relative;
+  bottom: 90px;
+  z-index: 900;
+}
+.transformers article {
+  padding: 180px 10px 10px 10px;
+  background-color: #3a83bb;
+  bottom: 280px;
+  z-index: 850;
+  position: relative;
+}
+.transformers header {
+  font-family: 'Oswald', sans-serif;
+  text-align: justify;
+  border-bottom: 10px solid #619bc7;
+  text-transform: uppercase;
+}
+.transformers h1 {
+  font-family: 'Oswald', sans-serif;
+  font-size: 30px;
+  margin: 0;
+}
+.transformers h3 {
+  font-family: 'Oswald', sans-serif;
+  font-size: 20px;
+  margin: 0 0 10px 0;
+}
+.transformers-p {
+  border-bottom: 2px dashed black;
+}
+.rating {
+  font-family: 'Oswald', sans-serif;
+  border-top: 5px solid #3476a8;
+  border-bottom: 5px solid #3476a8;
+  padding: 10px 0;
+  text-transform: uppercase;
+  width: 100%;
+}
+.rating h1 {
+  display: inline;
+}
+/*progress[value] {
```

Comment:
When we are using a version control system like Git, there is no good reason to leave commented code.